src/main/groovy/com/morpheusdata/nutanix/prismelement/plugin/NutanixPrismElementPlugin.groovy

```diff
@@ -28,9 +29,15 @@ class NutanixPrismElementPlugin extends Plugin {
     @Override
     void initialize() {
         this.setName("Nutanix Prism Element Plugin")
+        def backupProvider = new NutanixPrismElementBackupProvider(this, morpheus)
+
         this.registerProvider(new NutanixPrismElementPluginCloudProvider(this,this.morpheus))
         this.registerProvider(new NutanixPrismElementPluginProvisionProvider(this,this.morpheus))
         this.registerProvider(new NutanixPrismElementPluginNetworkPoolProvider(this, this.morpheus))
+        this.registerProvider(backupProvider)
+
+        pluginProviders.put(backupProvider.code, backupProvider)
```

Comment:
`this.registerProvider(..)` and `pluginProviders.put(..)` are the same thing, remove duplicate plugin registration.